plugins/modules/gcp_compute_instance_template.py

```diff
@@ -472,6 +473,10 @@
             elements: str
             required: false
             type: list
+  zone:
+    description:
+    - A reference to the zone where the disk type resides
+    type: str
```

Comment:
It would be great to put a `required: false` like the other parameters, but I can add that in as I merge the PR.

Reply:
I added `required: false` property